### PR TITLE
feat: weighted avg support in aggregator plugin

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

**Review by Hardikl — APPROVED** [review]
Would be good to add one test in aggregator_test for this.

File: cmd/poller/plugin/aggregator/aggregator.go [hunk 1]
```diff
@@ -209,11 +211,28 @@ func (a *Aggregator) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
...
+					opsKey := objMetric.GetComment()
+					if opsMetric = data.GetMetric(opsKey); opsMetric == nil {
+						a.Logger.Warn().Msgf("metric [%s] not found in [%s] response", opsKey, rule.label)
+						continue
+					}
+					if opsValue, ok = opsMetric.GetValueFloat64(instance); !ok {
+						continue
+					}
+					if err = objMetric.AddValueFloat64(objInstance, opsValue*value); err != nil {
+						a.Logger.Error().Stack().Err(err).Msgf("add value [%s] [%s]:", key, objName)
```
Comment (cgrinds) [inline cmd/poller/plugin/aggregator/aggregator.go:225]: minor: we've been removing unnecessary Stack() calls in error messages

File: cmd/poller/plugin/aggregator/aggregator.go [hunk 2]
```diff
@@ -209,11 +211,28 @@ func (a *Aggregator) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
...
+						continue
+					}
+					rule.counts[objKey][key] += opsValue
+				} else {
+					if err = objMetric.AddValueFloat64(objInstance, value); err != nil {
+						a.Logger.Error().Stack().Err(err).Msgf("add value [%s] [%s]:", key, objName)
```
Comment (cgrinds) [inline cmd/poller/plugin/aggregator/aggregator.go:231]: minor: we've been removing unnecessary Stack() calls in error messages

**Review by cgrinds — COMMENTED** [review]
nice job!

**Comment by cgrinds** [comment]
Should update `cmd/tools/generate/counter.yaml` with node_volume_read_latency, node_volume_write_latency, node_volume_other_latency

**Review by cgrinds — APPROVED** [review]

**Pull request merged**